src/ansys/api/pyensight/VERSION

```diff
@@ -1 +1 @@
-0.4.0dev0
+0.4.1
```

Comment:
@randallfrank just to follow the process here, could you please keep the version as 0.4.0dev0 in main, and then we make a new release?

I believe we just need to cherry pick the missing commits (comprising this one) in the branch release/0.3, update release/0.3 to version v0.3.5 and then make a new release out of it

Reply:
The idea is that we will make the 0.4.0 release when we will release the PyEnSight version 0.8.0, that will be when we will be asked to provide a package for the release 24.2 meta package

Reply:
@randallfrank  I fixed it for you. I want this PR in as much as you ahahhaa